src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/Decoder.cs

```diff
@@ -23,31 +23,31 @@ internal Decoder()
 
         private void InitializeDecoder()
         {
-            State = BrotliNative.BrotliDecoderCreateInstance();
-            if (State == IntPtr.Zero)
+            _state = BrotliNative.BrotliDecoderCreateInstance();
```

Comment:
Is "State" an instance of a native decoder? If so, calling it "state" seems odd.